src/routes/wallet/index.tsx

```diff
@@ -63,16 +64,14 @@ export default class Wallet extends React.Component<WalletProps, any> {
     return (
       <AppLayout section="wallet">
         <Web3SubmitWarning />
-
+        My address: <HexString>{this.props.account.address}</HexString>
```

Comment:
this shortens the address like `0x1234...aabb`, so it can not be copied. little unfortunate. maybe copy address button (separate issue).